Confus/Common/TeamScore.cpp

```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <memory>
+#include <RakNet/BitStream.h>
+#include <RakNet\MessageIdentifiers.h>
+#include <string>
+
+#include "TeamScore.h"
+#define DEBUG_CONSOLE
+#include "Debug.h"
+
+namespace ConfusServer 
+{
+    void TeamScore::setConnection(Networking::Connection* a_Connection)
+    {
+        m_Connection = a_Connection;
+    }
+
+    void TeamScore::sendScoreToClients()
+    {
+        RakNet::BitStream bitStream;
+        bitStream.Write(static_cast<RakNet::MessageID>(Networking::Connection::EPacketType::ScoreUpdate));
+
+        bitStream.Write(m_RedTeamScore);
+		bitStream.Write(m_BlueTeamScore);
+
+        m_Connection->sendPacketToAllClients(bitStream);
+    }
+
+    void TeamScore::sendWin()
+    {
+        //TODO: Send win to clients
+    }
+
+    void TeamScore::teamScoredPoint(ETeamIdentifier a_TeamScored, int a_IncreasementValue)
+    {
+        if(a_TeamScored == ETeamIdentifier::TeamBlue)
+        {
+            m_BlueTeamScore += a_IncreasementValue;
+        } 
+        else if(a_TeamScored == ETeamIdentifier::TeamRed)
+        {
+            m_RedTeamScore += a_IncreasementValue;
+        }
+        if(teamHasWon(a_TeamScored))
+        {
+            std::cout << "Team " << static_cast<int>(a_TeamScored) << " was the winner!" << std::endl;
+            sendWin();
+        }
+        sendScoreToClients();
+    }
+
+    int TeamScore::getPointsOfTeam(ETeamIdentifier a_Team) 
+    {
+        int scoreAmount = 0;
+        switch(a_Team)
+        {
+        case ETeamIdentifier::TeamBlue:
+            scoreAmount = m_BlueTeamScore;
+            break;
+        case ETeamIdentifier::TeamRed:
+            scoreAmount = m_RedTeamScore;
+            break;
+        default:
+            std::cout << "Error: Team does not exist.";
```

Comment:
I'd rather not have errors written to the standard output stream (i.e. cout). Perhaps throw an exception instead or something alike?